src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -138,7 +138,8 @@ public function response($path, $name = null, array $headers = [], $disposition
     {
         $response = new StreamedResponse;
 
-        $disposition = $response->headers->makeDisposition($disposition, $name ?? basename($path));
+        $target_name = $name ?? basename($path);
```

Comment:
`$target_name` camelCase 😃

Reply:
@fragkp `continuous-integration/styleci/pr` says that everything is ok. camelCase is this private requirement?

Reply:
I'm wondering why stylci is not failing. camelCase is the default for variable names in laravel. You could check this in all other files.

Reply:
StyleCI cannot fail for those cases, because it is not possible to fix such cases without breaking dynamic code. There are also various special cases where snake case is needed.

Reply:
Okay, I fixed it.
But need to fix the automatic codestyle verification.
External developers don't know your local codestyle rules.

Reply:
> But need to fix the automatic codestyle verification.

As I said, it is not broken. It is inherently not possible to safely rename variables.